jetty-core/jetty-fcgi/jetty-fcgi-client/src/main/java/org/eclipse/jetty/fcgi/parser/ResponseContentParser.java

```diff
@@ -187,7 +187,7 @@ public void parsedHeader(HttpField httpField)
             catch (Throwable x)
             {
                 if (LOG.isDebugEnabled())
-                    LOG.debug("Exception while invoking listener {}", listener, x);
+                    LOG.atDebug().setCause(x).log("Exception while invoking listener {}", listener);
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.